### DISCUSSION TITLE
Is there a way to decrease fps?

### DISCUSSION THREAD

**Discussion by reignerdeustcher** [diasurgical/DevilutionX#6971] [Q&A]
### Operating System

Windows x64

### DevilutionX version

1.5.2

### Describe

The game runs so smooth with this, any chance to change the fps for the original diablo feel?

### To Reproduce

1

**Comment by StephenCWills** [comment]
Issues is not the right place for this kind of question, and certainly not using the bug report template. Converting to a discussion.

**Comment by StephenCWills** [comment]
There's no way to do this in the app. If you have an NVIDIA GPU, NVIDIA Control Panel can limit the FPS of the app.

![image](https://github.com/diasurgical/devilutionX/assets/9203145/fc97ce2d-d1e8-4962-a608-c60af87c4fb0)

**Comment by stephematician** [comment]
For those with __linux__ there is also the option of using [MangoHud](https://github.com/flightlessmango/MangoHud):

```sh
mangohud --dlsym devilutionx-linux-x86_64.appimage
```

Then you can configure __MangoHud__ in any number of the usual ways, e.g. with a `.config/MangoHud/devilutionx.conf`:

```ini
fps_limit=60
```
